src/react-apps/applications/runtime/src/App.tsx

```diff
@@ -71,7 +71,8 @@ export default function() {
     <MuiThemeProvider theme={theme}>
       <Route path={'/'} exact={true} component={Instantiate} />
       <Route path={'/partyselection'} exact={true} component={PartySelection} />
-      <Route path={'/instance/:partyId/:instanceGuid'} component={FormFiller} />
+      <Route path={'/partyselection/:errorCode'} exact={true} component={PartySelection} />
```

Comment:
Ref linje 73, kunne man ha brukt optional https://stackoverflow.com/a/35604855 ?

Reply:
Cool. Using it.